packages/api-client/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@cumulus/api-client",
+  "version": "1.19.0",
+  "description": "API client for working with the Cumulus archive API",
+  "keywords": [
+    "GIBS",
+    "CUMULUS",
+    "NASA"
+  ],
+  "engines": {
+    "node": ">=10.16.3"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/api-client",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nasa/cumulus"
+  },
+  "scripts": {
+    "build": "true",
+    "lint": "eslint .",
+    "test": "ava",
+    "test-coverage": "nyc ava",
+    "test-ci": "npm run lint"
+  },
+  "ava": {
+    "files": [
+      "tests/**"
+    ],
+    "verbose": true
+  },
+  "nyc": {
+    "exclude": [
+      "tests",
+      "test"
+    ]
+  },
+  "author": "Cumulus Authors",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@cumulus/aws-client": "1.19.0",
+    "@cumulus/logger": "1.19.0",
+    "lodash.clonedeep": "^4.5.0",
+    "p-retry": "^2.0.0"
+  },
+  "devDependencies": {
+    "@cumulus/common": "1.19.0",
```

Comment:
Is this actually used?

Reply:
https://github.com/nasa/cumulus/pull/1541/commits/0f168f8bd1d41d4905f1b17c6f0973354f891afa